### ISSUE TITLE
Unification towards Mythic Upgrades for ruby is problematic for MI

### ISSUE THREAD

**Issue by truepurpledino** [TeamAOF/All-of-Fabric-7#56]
On previous versions of AOF, ruby was generally acquired by tech reborn ruby ores which spawned in the overworld quite frequently.
In AOF7 1.0.0, it seems the only way to get ruby for chromium is through Mythic Upgrades ruby, which is a very very rare ore, or with redstone centrifuging, which is a lot more costly in terms of energy compared to ore processing. I feel like this change is illogical for MI progression. Ruby ore from Mythic Upgrades is rarer than netherite and biome specific.
I would recommend adding back Tech Reborn ruby ore to the quarry or overworld generation, but separating the uses of Mythic Upgrades ruby from Tech Reborn ruby or buffing the current ways to producing ruby dust (for example lowering the redstone centrifuging recipe total EU, adding more ruby dust to the output or changing the current gold drill quarry output chance, which is right now 1 ruby per five drills).

**Comment by iteathen** [comment]
Yes, there is something of a balance problem around ruby dust. Just thinking about it offhand, doubling the output of ruby dust might address this without affecting the balance of Mythic Upgrades.
